src/Symfony/Component/PropertyAccess/Tests/Fixtures/ExtendedUninitializedProperty.php

```diff
@@ -13,5 +13,4 @@
 
 class ExtendedUninitializedProperty extends UninitializedProperty
 {
-
```

Comment:
Note to reviewers - this was an unintended side effect of running CS Fixer, should happen anyway.